docs/breakpad_reporting.rst

```diff
@@ -163,20 +165,22 @@ development instance. There are a few options:
 
 4. On Linux, you can crash processes using ``kill``::
 
-       kill -ABRT $(pidof firefox)
+       kill -SIGFPE $(pidof firefox)
 
 
 You can capture a raw HTTP POST this way:
 
-1. Run ``nc -l localhost 8000 > http_post.raw`` in one terminal
+1. Run ``nc -l localhost 8000 > http_post.raw`` in one terminal.
 
-2. Run ``MOZ_CRASHREPORTER_URL=http://localhost:8000/submit firefox`` in a second terminal
+2. Run ``MOZ_CRASHREPORTER_URL=http://localhost:8000/submit firefox`` in a
+   second terminal.
 
-3. Run ``ps -aef``, find the firefox process id and then do ``kill -ABRT <PID>`` in a
-   third terminal
+3. Run ``ps -aef | grep firefox``, find the process id for the main firefox
+   process or a content process, and then do ``kill -SIGFPE <PID>`` in a third
+   terminal.
 
-4. The Firefox process will crash and the crash report dialog will pop up. Make sure
-   to submit the crash, then click on "Quit Firefox" button
+4. The Firefox process will crash and the crash report dialog will pop up.
+   Make sure to submit the crash, then click on "Quit Firefox" button.
```

Comment:
This process doesn't work for me using my normal Nightly build because crash reporting is disabled over insecure contexts. Can that be disabled?

Reply:
I have no idea. I don't think I've had that problem with release or nightly. I think we should push off figuring this out to a separate PR/bug.